doc/src/manual/integers-and-floating-point-numbers.md

```diff
@@ -671,11 +671,18 @@ where syntactic conflicts arise:
     `0` multiplied by the variable `xff`.
   * The floating-point literal expression `1e10` could be interpreted as the numeric literal `1` multiplied
     by the variable `e10`, and similarly with the equivalent `E` form.
+  * The 32-bit floating-point literal expression `1.5f22` could be interpreted as the numeric literal
+    `1.5` multiplied by the variable `f22`.
 
-In both cases, we resolve the ambiguity in favor of interpretation as numeric literals:
+In all cases, we resolve the ambiguity in favor of interpretation as numeric literals:
 
   * Expressions starting with `0x` are always hexadecimal literals.
   * Expressions starting with a numeric literal followed by `e` or `E` are always floating-point literals.
+  * Expressions starting with a numeric literal followed by `f` are always 32-bit floating-point literals.
+    Unlike `E`, which is equivalent to `e` in numeric literals for historical reasons, `F` is just another
```

Comment:
How about de-denting this so that it's just text after the bullet points. I feel like including this in the last bullet point reduces clarity whereas including it in the text after wards as additional explanation works well.

Reply:
Do you prefer just a regular text or inside a note block?